playbooks/cluster-up.yaml

```diff
@@ -6,8 +6,7 @@
   gather_facts: False
 
   tasks:
-  - name: "Checking ssh-agent is running and '{{ ssh_key }}' key is loaded"
-    shell: "ssh-add -l | grep {{ ssh_key | quote }}"
+  - import_tasks: tasks/check-ssh.yaml
```

Comment:
since this is only ever called once, we don't really need to extract it to a separate file.  the ec2_instance and ec2 tasks for stopping instances could be though.